src/refer-doc.ts

```diff
@@ -1,16 +1,13 @@
 /* Copyright © 2022 Seneca Project Contributors, MIT License. */
 
-
 const docs = {
-
   actCreateEntry: {
-    desc: 'Create referral entry.',
+    desc: "Create referral entry.",
```

Comment:
Understood, thank you for the reference. I managed to make some recent updates. And move back to use "single quotes", instead of double. When you can, take a look again, please.